.github/workflows/e2e.yml

```diff
@@ -36,7 +36,10 @@ jobs:
         sudo apt install ffmpeg  # for local Whisper tests
 
     - name: Install Haystack
-      run: pip install .[dev,audio] langdetect transformers[torch,sentencepiece]==4.35.2 'sentence-transformers>=2.2.0' pypdf tika 'azure-ai-formrecognizer>=3.2.0b2'
+      run: pip install .[dev]
```

Comment:
At this point we can put `.[dev]` in the requirements file and spare one job step

Reply:
done!